### PR TITLE
Added programming tip that made my programming workflow easier!

### PR DESCRIPTION
**Pull request by Master-Keshav**
This solves issue #27 

Modularizing the code is a good practice that leads to more maintainable and scalable software. I hope this will help.

### CODE REVIEW

File: app/messages/programming-tip [hunk 1]
```diff
@@ -0,0 +1,24 @@
+By: Keshav Vyas
+
+
```
Comment (SavvasStephanides) [inline app/messages/programming-tip:3]: Please start your message at the 3rd line

File: app/messages/programming-tip [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+5) Testing: Smaller modules are typically easier to test. We can write unit tests for individual modules to ensure they work correctly.
+
+6) Scaling: As our project grows, modular code is easier to scale. We can add or update modules without affecting the entire application.
+
+*******
+
+For example, in Python, we can create separate Python files for different functionalities and import them as modules in our main script. In JavaScript, we can use ES6 modules or CommonJS to achieve a similar modular structure.
+
+Modularizing the code is a good practice that leads to more maintainable and scalable software.
```
Comment (SavvasStephanides) [inline app/messages/programming-tip:24]: Your message is a bit too long. Please reduce your message to under 400 characters. Thank you!

**Review by SavvasStephanides — CHANGES_REQUESTED** [review]

File: app/messages/programming-tip (hunk 1 above)
Comment (Master-Keshav) [reply app/messages/programming-tip]: Done

Comment (Master-Keshav) [reply app/messages/programming-tip]: Done @SavvasStephanides

**Review by SavvasStephanides — APPROVED** [review]
Thank you so much @Master-Keshav! Your message looks great! 🎉

Your message should appear here once it deploys:

https://just-say-something.vercel.app/message/programming-tip

Congratulations on your first open source contribution!

**Comment by Master-Keshav** [comment]
> Thank you so much @Master-Keshav! Your message looks great! 🎉
> 
> Your message should appear here once it deploys:
> 
> https://just-say-something.vercel.app/message/programming-tip
> 
> Congratulations on your first open source contribution!

Thanks alot @SavvasStephanides

**Pull request merged**